go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/prometheus/client_golang v1.14.0
 	github.com/rs/cors v1.8.3
-	github.com/sassoftware/relic v0.0.0-20210427151427-dfb082b79b74
+	github.com/sassoftware/relic v7.2.1+incompatible
```

Comment:
Looking at the git history, the dfb082b79b74bf792c0bfd7aca6e22bea6156a85 commit is between `7.2.6` and `7.3.0`, i.e. this is a downgrade.

Was that intentional?